.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    'extends': [
+        'eslint:recommended',
+        'plugin:react/recommended',
+        'eslint-config-jitsi'
+    ],
```

Comment:
Do you have to extend 'eslint:recommended' if it's already defined in eslint-config-jitsi?

Reply:
fixed